### ISSUE TITLE
Big Sur Safari WebExtensions API support

### ISSUE THREAD

**Issue by damo78** [reduxjs/redux-devtools#678]
This is just a question so hope it is okay to post here.

I've noticed that in Apple's Big Sur release they've added WebExtensions API support to Safari and suggested for developers it's an easy port.  Is there a chance of having a Redux Dev Tools extension in Safari now?

**Comment by Methuselah96** [comment]
Yeah, it sounds like it should be possible. I'll plan to include a Safari extension release for 3.0 if it truly is an easy port.

**Comment by Uhucream** [comment]
According to [Apple's document](https://developer.apple.com/documentation/safariservices/safari_web_extensions/assessing_your_safari_web_extension_s_browser_compatibility), Safari 14 doesn't correspond to devtools.

So I think it is impossible.

**Comment by trytuna** [comment]
Any update on this?

**Comment by dvaldivia** [comment]
@Methuselah96 any luck? anything we can help with? I can gladly try and port some of the needed code if you point me in the right direction

**Comment by markvl91** [comment]
In Safari 17 the extension generated by using the `xcrun safari-web-extension-converter` tool on the non-prod Firefox version of the extension only requires removing the `chrome.notifications.onClicked` reference in the background page script in order to get a fully working Redux DevTools. 

So close, but I can image the distribution overhead that Apple requires for Safari extensions nowadays still makes this quite the hassle...
